build/cli/src/arg/gui2.rs

```diff
@@ -15,16 +16,32 @@ source_args_hlp!(Gui2, "gui2", BuildInput);
 #[derive(Args, Clone, Copy, Debug, PartialEq)]
 pub struct BuildInput {}
 
+#[derive(ArgEnum, Clone, Copy, Debug, PartialEq)]
+pub enum TestType {
+    /// Run unit tests in watch mode
+    Unit,
+    /// Run E2E tests
+    E2E,
+    /// Run all tests without watching nor HTML reports.
```

Comment:
Producing HTML reports for failing tests and adding them to the CI artifacts could maybe be useful. Something to check if it's worth the effort.

Reply:
Let's do it in a separate PR